website/src/components/SearchPage/DownloadDialog/SequenceFilters.tsx

```diff
@@ -39,6 +39,10 @@ export class FieldFilter implements SequenceFilter {
     private readonly hiddenFieldValues: FieldValues;
     private readonly schema: ConsolidatedMetadataFilters;
 
+    /**
+     * @param lapisSearchParamters The filters that are set
+     * @param hiddenFieldValues Key-value pairs that - if those are found - they will not be rendered.
```

Comment:
```suggestion
     * @param hiddenFieldValues Key-value pairs that - if even if they are found in the parameters - will not be rendered.
```

If this is what is meant